service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/history/SealedOrderHistoryService.java

```diff
@@ -60,13 +61,18 @@ public class SealedOrderHistoryService {
     private final ManageOrdersClosedCaseFieldGenerator manageOrdersClosedCaseFieldGenerator;
 
     public Map<String, Object> generate(CaseData caseData) {
-        List<Element<GeneratedOrder>> pastOrders = caseData.getOrderCollection();
         ManageOrdersEventData manageOrdersEventData = caseData.getManageOrdersEventData();
         List<Element<Child>> selectedChildren = childrenSmartSelector.getSelectedChildren(caseData);
         List<Element<Other>> selectedOthers = othersService.getSelectedOthers(caseData);
 
-        DocumentReference sealedPdfOrder = orderCreationService.createOrderDocument(caseData, OrderStatus.SEALED, PDF);
-        DocumentReference plainWordOrder = orderCreationService.createOrderDocument(caseData, OrderStatus.PLAIN, WORD);
+        DocumentReference order;
+        DocumentReference plainWordOrder;
+        if (OrderSourceType.MANUAL_UPLOAD == manageOrdersEventData.getManageOrdersType().getSourceType()) {
```

Comment:
Might be good to add a comment to say why we're doing manual upload differently

Reply:
sure, added some comment